test/acceptance/spec/10-search.js

```diff
@@ -26,25 +26,25 @@ describe('Search Page', function() {
   });
 
   describe('submitting a valid query', function () {
-    beforeEach(function () {
-      browser.setValue('input[name="surname"]', 'Churchill');
-      browser.setValue('input[name="forenames"]', 'Winston');
-      browser.setValue('input[name="dob"]', '30/11/1874');
-    });
-
     describe('that returns no records', function () {
       beforeEach(function () {
+        browser.setValue('input[name="surname"]', 'Churchil');
```

Comment:
I believe this was deliberate. (I'm reviving some of Tim's code.)
As far as I can tell this is to simulate a typo.
If there search was for "Churchill" the test would fail when run as an end-to-end test but not as an acceptance test against a mock API.